mode/examples/Topics/File IO/SaveFrames/SaveFrames.pyde

```diff
@@ -0,0 +1,62 @@
+"""
+Save Frames
+by Daniel Shiffman.
+
+This example demonstrates how to use saveFrame() to render
+out an image sequence that you can assemble into a movie
+using the MovieMaker tool.
+"""
+
+# A boolean to track whether we are recording are not
+recording = False
+
+
+def setup():
+    size(640, 360)
+
+
+def draw():
+    background(0)
+
+    # An arbitrary oscillating rotating animation
+    # so that we have something to render
+    for a in xrange(0, TWO_PI, 1):
```

Comment:
This shouldn't even run; I don't think you can use a foating point number in a range statement. Have you run these programs?